config/default/tox.ini.j2

```diff
@@ -0,0 +1,35 @@
+[tox]
+envlist =
+    format
+    lint
+
+[testenv]
+py_files = find %(dist_path)s -name "*.py"
```

Comment:
This is missing any Python files outside of the `plone` directory, for example `setup.py`.
Can you try `git ls-files "*.py"` ? That works for me.

Reply:
Oh, that looks great 🤩 ✨ 👍🏾  we can simplify the `xargs` call with it